src/control/cmd/dmg/storage_query.go

```diff
@@ -150,3 +151,46 @@ func (cmd *listPoolsQueryCmd) Execute(_ []string) error {
 	}
 	return cmd.makeRequest(ctx, req, control.PrintWithVerboseOutput(cmd.Verbose))
 }
+
+// usageQueryCmd is the struct representing the scan storage subcommand.
+type usageQueryCmd struct {
+	logCmd
+	ctlInvokerCmd
+	hostListCmd
+	jsonOutputCmd
+	//Verbose    bool `short:"v" long:"verbose" description:"List SCM & NVMe device details"`
+}
+
+// Execute is run when usageQueryCmd activates.
+//
+// Queries NVMe and SCM usage on hosts.
+func (cmd *usageQueryCmd) Execute(_ []string) error {
+	ctx := context.Background()
+	req := &control.StorageScanReq{}
+	req.SetHostList(cmd.hostlist)
+	resp, err := control.StorageScan(ctx, cmd.ctlInvoker, req)
+
+	if cmd.jsonOutputEnabled() {
+//		if cmd.Verbose {
+//			cmd.log.Info("--verbose flag ignored if --json specified")
+//		}
```

Comment:
I noted this on a different PR, but this output won't be seen if JSON output is enabled: https://github.com/daos-stack/daos/blob/master/src/control/cmd/dmg/main.go#L227

Reply:
fixed a couple of occurrences of this in other files